src/ClusterBootstrap/ctl.py

```diff
@@ -253,6 +254,67 @@ def render_template_or_dir(config, args):
         utils.render_template(src, dst, config)
 
 
+def maintain_db(config, args):
+    """
+    push/pull a table to/from DB
+    """
+    subcommand = args.nargs[0]
+    assert subcommand in ["pull", "push", "connect"], "invalid action."
+    host = config["mysql_node"]
+    user = config["mysql_username"]
+    password = config["mysql_password"]
+    if subcommand == "connect":
+        os.system("mysql -h {} -u {} -p{}".format(host, user, password))
+    else:
+        database = "DLWSCluster-{}".format(config["clusterId"])
+        table_name = args.nargs[1]
+        assert table_name in ["vc", "acl"], "invalid table."
```

Comment:
We should list what tables are valid in this operation

Reply:
is there a list about what tables should be valid? are there any tables no longer in use.?